FV3/wrapper/lib/Makefile

```diff
@@ -2,29 +2,28 @@ SHELL = /bin/sh
 
 TEMPLATES = $(shell ls ../templates)
 PROPERTIES_FILES = ../fv3gfs/wrapper/dynamics_properties.json ../fv3gfs/wrapper/physics_properties.json
+include $(MAKE_INCLUDE)
 
 # Use pkg-config to get flags for fv3
 # If we run without PKG_CONFIG_ALLOW_SYSTEM_CFLAGS,
 # the -I/usr/include flag will be missing
-FFLAGS += $(shell PKG_CONFIG_ALLOW_SYSTEM_CFLAGS=1 pkg-config fv3 --cflags)
 FC ?= mpif90
 
 .PHONY: clean cleanall
 
 all: coupler_lib.o dynamics_data.o physics_data.o flagstruct_data.o $(TEMPLATES)
 
 %.o: %.F90
-	$(FC) $(FFLAGS)  -c $< -o $@
+	$(FC) $(CPPDEFS) $(CPPFLAGS) $(FPPFLAGS) $(FFLAGS) $(OTHERFLAGS) $(OTHER_FFLAGS) $(ESMF_INC) -c $< -o $@
 
 %.o: %.f90
-	$(FC) $(FFLAGS)  -c $< -o $@
+	$(FC) $(CPPDEFS) $(CPPFLAGS) $(FPPFLAGS) $(FFLAGS) $(OTHERFLAGS) $(OTHER_FFLAGS) $(ESMF_INC) -c $< -o $@
 
 physics_data.o: physics_data.F90 dynamics_data.o
 
 coupler_lib.o: coupler_lib.F90 dynamics_data.o
 
 flagstruct_data.o: flagstruct_data.F90
-	$(FC) $(CPPDEFS) $(CPPFLAGS) $(FPPFLAGS) $(FFLAGS) $(OTHERFLAGS) $(OTHER_FFLAGS) $(ESMF_INC) -c $< -o $@
```

Comment:
Do you need to delete the flagstruct_data.o target so it uses the wildcard target instead? If so, how is this not picked up by tests?

Reply:
This is  how pattern rules work. See this example
```
$ cat Makefile
%.txt: %.out
        cp $< $@


stem.txt: stem.out another-file

another-file:
        touch another-file

stem.out:
        touch stem.out
$ make
touch stem.out
touch another-file
cp stem.out stem.txt
```

Reply:
OK, interesting. I think also you could delete this make target in this case, since it's covered by the wildcard. But good as-is, resolving.